internal/failover/ping_default.go

```diff
@@ -0,0 +1,18 @@
+//go:build !windows
+
+package failover
+
+import (
+	"net"
```

Comment:
missing imports
```
"github.com/go-errors/errors"
"golang.org/x/net/icmp"```

Reply:
Missed this one as `go build` didn't touch this file on my computer. 😇 Fixing...